pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -9,6 +9,7 @@
 
 
 @pytest.mark.filterwarnings("ignore:Sparse:FutureWarning")
+@pytest.mark.filterwarnings("ignore:Series.to_sparse:FutureWarning")
```

Comment:
where is this coming from?

Reply:
From the Series.to_sparse calls in those tests.